dapp-manager/src/main/kotlin/jp/co/soramitsu/dapp/manager/registration/DappAccountRegistrator.kt

```diff
@@ -0,0 +1,172 @@
+                // Here we need our own key to sign this stuff
+                IrohaTransaction(
+                    newUserAccountId,
+                    ModelUtil.getCurrentTime(),
+                    listOf(
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_set_my_quorum_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_add_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            brvsAccountId,
+                            Primitive.GrantablePermission.can_remove_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_set_my_quorum_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_add_my_signatory_VALUE
+                        ),
+                        IrohaCommand.CommandGrantPermission(
+                            irohaConsumer.creator,
+                            Primitive.GrantablePermission.can_remove_my_signatory_VALUE
+                        )
+                    )
+                ),
+                // Also we need to give permissions from requests account perspective
+                IrohaTransaction(
+                    requestAccountId,
+                    ModelUtil.getCurrentTime(),
+                    listOf(
+                        IrohaCommand.CommandGrantPermission(
+                            newUserAccountId,
+                            Primitive.GrantablePermission.can_set_my_account_detail_VALUE
+                        )
+                    )
+                ),
+                // Finally we need to add user's original pub key to the signatories list
+                // But we need to increase user's quorum to prohibit transactions using only user's key
+                // Several moments later BRVS react on the createAccountTransaction and
+                // user's quorum will be set as 1+2/3 of BRVS instances for now
+                IrohaTransaction(
+                    irohaConsumer.creator,
+                    ModelUtil.getCurrentTime(),
+                    listOf(
+                        IrohaCommand.CommandAddSignatory(
+                            newUserAccountId,
+                            pubkey
+                        ),
+                        IrohaCommand.CommandSetAccountQuorum(
+                            newUserAccountId,
+                            2
```

Comment:
What is 2? Looks like magic constant.

Reply:
same logic as our `notary-registration` service, default Iroha quorum is 2 for a newcomer